internal/actions/pr.go

```diff
@@ -69,7 +73,7 @@ func CreatePullRequest(ctx context.Context, repo *git.Repo, client *gh.Client, o
 		//       break if the upstream name is not the same name as the local
 		upstream, err := repo.RevParse(&git.RevParse{
 			SymbolicFullName: true,
-			Rev:              "HEAD@{u}",
+			Rev:              fmt.Sprintf("%s@{u}", opts.BranchName),
```

Comment:
this is an unrelated bug: we would always push whatever branch was currently checked out rather than the target branch.

Bad behavior (that's now fixed):
```
$ av stack submit
Creating pull request for branch 2022-09-12-one:
  - pushing branch to GitHub (origin/2022-09-12-three)
...
```
(note that the `pushing branch to` bit is wrong)